changelogs/fragments/8979-keycloak_group-fix-subgroups.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - keycloak_group - fix crash caused in subgroup creation. The crash was caused by a missing or empty ``subGroups`` property in Keycloak ≥23 (https://github.com/ansible-collections/community.general/issues/8788, https://github.com/ansible-collections/community.general/pull/8979).
```

Comment:
@felixfontein will the changelog builder handle special unicode characters? If not, then we might need to change the `≥` to `>=`.

Reply:
The changelog generator doesn't care (as long as everything is UTF-8 encoded), and both RST and MarkDown should handle Unicode fine.